.github/workflows/auto-fix-issues.yml

```
@@ -0,0 +1,89 @@
+              username: context.payload.comment.user.login,
+            });
+            const allowed = ["admin", "maintain", "write"].includes(data.permission);
+            core.setOutput("allowed", allowed ? "true" : "false");
+
+      - name: Parse app command (fix/rewrite/refactor/test/docs/pr)
+        if: steps.perm.outputs.allowed == 'true'
+        id: parse
+        run: |
+          BODY=$(cat <<'EOF'
+          ${{ github.event.comment.body }}
+EOF
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)